### PR TITLE
[DPE-2634] First version of the release library pipeline

### PR DESCRIPTION
**Pull request by welpaolo**
This PR enable the possibility to release jar file with the central uploader.

I will remove `on_pull_requests` when addressing PR feedbacks if any.

### CODE REVIEW

**Comment by deusebio** [comment]
@welpaolo what's the fate of this PR?

**Comment by welpaolo** [comment]
In review now

File: uploader/utils.py [hunk 1]
```diff
@@ -109,6 +115,38 @@ def check_new_releases(
...
+        for filename in os.listdir(f"{output_directory}/{release_directory}"):
+            if fnmatch.fnmatch(filename, library_pattern):
+                # get library name without extension
+                library_name = filename.rsplit(".", 1)[0]
+                break
+        print(f"Library name: {library_name}")
+        assert library_name
+        # check them against tags in Github
+        related_tags = get_library_tags(repository_owner, project_name, library_name)
+        print(f"Related tag: {related_tags}")
```
Comment (deusebio) [inline uploader/utils.py:139]: **todo** weren't we using loggers?

**Review by deusebio — APPROVED** [review]
Looks fine!

**Review by Mehdi-Bendriss — APPROVED** [review]

File: uploader/utils.py (hunk 1 above)
Comment (welpaolo) [reply uploader/utils.py]: Done

**Pull request merged**